mlflow/server/js/src/model-registry/components/ModelVersionPage.js

```diff
@@ -70,12 +72,12 @@ class ModelVersionPage extends React.Component {
     const toStage = activity.model_registry_data.to_stage;
     if (activity.type === ActivityTypes.APPLIED_TRANSITION) {
       this.props
-        .updateModelVersionApi(
+        .transitionModelVersionStageApi(
           modelName,
           version,
           toStage,
-          undefined,
-          this.updateModelVersionRequestId,
+            false,  // TODO(andy.chow): once we have a dialog calling this we can pass in this param
+          this.transitionModelVersionStageRequestId,
```

Comment:
This UUID needs to be defined [here](https://github.com/mlflow/mlflow/pull/2529/files#diff-95e151225f64de4822a5a6fad2407db5L40).